main/store/migrations/index.js

```diff
@@ -83,7 +83,7 @@ const migrations = {
       type: 'ethereum',
       symbol: 'MATIC',
       name: 'Polygon',
-      explorer: 'https://explorer.matic.network',
+      explorer: 'https://polygonscan.com',
```

Comment:
If we want to apply this to everyone who currently has the old explorer set we will need to add a new migration, thoughts?

Reply:
we could, I wasn't sure that it was that important to warrant a migration but no good reason not to. I'll add one.